### PR TITLE
CIRC-7321 - check metrics for statsd_type count, and if so, treat it …

### PR DESCRIPTION
**Pull request by bdunavant**
…differently

### CODE REVIEW

**Comment by bdunavant** [comment]
This is actually for CIRC-7324

File: circonus/circonus.go [hunk 1]
```diff
@@ -262,26 +262,50 @@ func (c *Client) HandleStatsdAggregations(s string) string {
 		logger.Printf(logger.LvlError, err.Error())
 		return "" // TODO - return error
 	}
+	var statsdType string = ""
```
Comment (dhaifley) [inline circonus/circonus.go:265]: it is more idiomatic to just write: statsdType := ""

**Review by dhaifley — APPROVED** [review]
LGTM

**Review by dhaifley — APPROVED** [review]
LGTM

**Pull request merged**